frb_example/pure_dart/rust/src/api.rs

```diff
@@ -1156,6 +1156,7 @@ pub fn run_opaque(opaque: RustOpaque<HideData>) -> String {
     opaque.hide_data()
 }
 
+#[allow(clippy::redundant-clone)]
```

Comment:
Well I would suggest just fix that linter error

Reply:
The clone is the whole point of that test though.

Reply:
Oh I see. Then would be great to add a line of comment to explain it